cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -142,6 +142,67 @@ using rhs_iterator = strong_index_iterator<rhs_index_type>;
 
 namespace lexicographic {
 
+/**
+ * @brief Computes a weak ordering of two values with special sorting behavior.
+ *
+ * This relational comparator functor compares physical values rather than logical
+ * elements like lists, strings, or structs. It evaluates `NaN` as equivalent to other `NaN`s and
+ * greater than all other values.
```

Comment:
Something got copied incorrectly and this docstring is the same as the other one. See: https://github.com/rapidsai/cudf/pull/10870/#discussion_r888415494
```suggestion
 * This relational comparator functor compares physical values rather than logical
 * elements like lists, strings, or structs. It evaluates `NaN` as not less than, equal to, or
 * greater than other values and is IEEE-754 compliant.
```